akka-actor/src/main/scala/akka/actor/Address.scala

```diff
@@ -33,6 +33,14 @@ final case class Address private (protocol: String, system: String, host: Option
   def this(protocol: String, system: String) = this(protocol, system, None, None)
   def this(protocol: String, system: String, host: String, port: Int) = this(protocol, system, Option(host), Some(port))
 
+  def copy(
+      protocol: String = protocol,
+      system: String = system,
+      host: Option[String] = host,
+      port: Option[Int] = port) = {
```

Comment:
ok, so this is the same as already is generated by Scala 2?

Reply:
Yes, here we are enforcing the visibility that changed with Scala 3